testing/ui-tests/src/utils/pallet_metadata_test_runner.rs

```diff
@@ -106,18 +91,10 @@ impl PalletMetadataTestRunner {
 // are dropped too, to make sure that cleanup happens after tests are ran.
 impl Drop for PalletMetadataTestRunner {
     fn drop(&mut self) {
-        for i in 0..self.index {
-            if let Some(pallet) = self.metadata.pallets.get(self.index) {
-                if let Some(name_filter) = self.pallet_names.as_ref() {
-                    if !name_filter.contains(&pallet.name) {
-                        continue;
-                    }
-                }
-
-                let mut tmp_dir = std::env::temp_dir();
-                tmp_dir.push(format!("{TEST_DIR_PREFIX}{i}"));
-                std::fs::remove_dir_all(tmp_dir).expect("cannot cleanup temp files");
-            };
+        for idx in 0..self.index {
+            let mut tmp_dir = std::env::temp_dir();
+            tmp_dir.push(format!("{TEST_DIR_PREFIX}{idx}"));
+            let _ = std::fs::remove_dir_all(tmp_dir);
```

Comment:
Simplified because we only care about index and not whether we are iterating all or specific pallets.